sources/api/schnauzer/src/helpers.rs

```diff
@@ -2362,3 +2397,75 @@ mod test_etc_hosts_helpers {
         assert_eq!(result, "")
     }
 }
+
+#[cfg(test)]
+mod test_any_enabled {
+    use super::*;
+    use handlebars::RenderError;
+    use serde::Serialize;
+    use serde_json::json;
+    use std::collections::BTreeMap;
+
+    // A thin wrapper around the handlebars render_template method that includes
+    // setup and registration of helpers
+    fn setup_and_render_template<T>(tmpl: &str, data: T) -> Result<String, RenderError>
+    where
+        T: Serialize,
+    {
+        let mut registry = Handlebars::new();
+        registry.register_helper("any_enabled", Box::new(any_enabled));
+
+        let mut input_data = BTreeMap::new();
+        input_data.insert("input", data);
+
+        registry.render_template(tmpl, &input_data)
+    }
+
+    const TEMPLATE: &str = r#"{{#if (any_enabled input)}}enabled{{else}}disabled{{/if}}"#;
+
+    #[test]
+    fn test_any_enabled_with_enabled_elements() {
+        let result =
+            setup_and_render_template(TEMPLATE, &json!([{"foo": [], "enabled": true}])).unwrap();
+        let expected = "enabled";
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_any_enabled_without_enabled_elements() {
+        let result = setup_and_render_template(TEMPLATE, &json!([{"enabled": false}])).unwrap();
+        let expected = "disabled";
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_any_enabled_with_empty_elements() {
+        let result = setup_and_render_template(TEMPLATE, &json!([])).unwrap();
+        let expected = "disabled";
+        assert_eq!(result, expected);
+    }
+
+    #[test]
+    fn test_any_enabled_with_bool_flag_as_string() {
+        // Helper is only expected to work with boolean values
+        let result = setup_and_render_template(TEMPLATE, &json!([{"enabled": "true"}])).unwrap();
```

Comment:
Seems important for testing to verify that fields other than `enabled` have no effect.

```suggestion
        let result = setup_and_render_template(TEMPLATE, &json!([{"foo": [], "enabled": "true"}])).unwrap();
```

Reply:
This is being done on line 2429. Since this (line 2451) tests a negative case I don't know if it would add much value here. Thoughts?

Reply:
Sorry I missed that. This is good.